package.json

```diff
@@ -13,14 +13,11 @@
     "@testing-library/jest-dom": "^5.11.9",
     "@testing-library/react": "^11.2.5",
     "@testing-library/user-event": "^12.8.1",
-    "@types/jest": "^26.0.20",
-    "@types/node": "^14.14.31",
-    "@types/react": "^17.0.3",
-    "@types/react-dom": "^17.0.1",
     "@types/react-router-dom": "^5.1.7",
     "font-awesome": "^4.7.0",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
+    "react-route-props": "^1.0.4",
```

Comment:
No need for that one at all 😆 
`react-router` has already all the props you need!

Delete this module and in `routes.ts` add the following:
`import { RouteProps } from 'react-router'`